aptoide-ui/src/main/java/cm/aptoide/pt/aptoide_ui/theme/Theme.kt

```diff
@@ -380,6 +381,14 @@ val darkTypography = AppTypography(
   )
 )
 
+private val lightIcons = AppIcons(
+  ToolbarLogo = getToolbarLogo(textBlack, aptoideIconOrange, aptoideIconBackgroundWhite)
+)
+
+private val darkIcons = AppIcons(
+  ToolbarLogo = getToolbarLogo(textWhite, aptoideIconOrange, aptoideIconBackgroundWhite)
```

Comment:
The 'textWhite' that is only used in the AptoideLogo is different from the one in design. It should be '0xFFFFFFFF'